rfc6962-bis.xml

```diff
@@ -481,26 +474,119 @@ Section 8.2 of <xref target="RFC3447"/>) using a key of at least 2048 bits.
+        </t>
+        <t>
+          <spanx style="verb">type</spanx> is the type of the encapsulated data structure. (Note that <spanx style="verb">TransType</spanx> combines the v1 type enumerations <spanx style="verb">LogEntryType</spanx>, <spanx style="verb">SignatureType</spanx> and <spanx style="verb">MerkleLeafType</spanx>). Future revisions of this protocol may add new <spanx style="verb">TransType</spanx> values.
+        </t>
+        <t>
+          <spanx style="verb">data</spanx> is the encapsulated data structure. The various structures named with the <spanx style="verb">DataV2</spanx> suffix are defined in later sections of this document.
+        </t>
+        <t>
+          <spanx style="verb">item_extension_type</spanx> identifies a single extension from the IANA registry in <xref target="item_extension_types"/>.
+        </t>
+        <t>
+          The interpretation of the <spanx style="verb">item_extension_data</spanx> field is determined solely by the value of the <spanx style="verb">item_extension_type</spanx> field. Each document that registers a new <spanx style="verb">item_extension_type</spanx> must describe how to interpret the corresponding <spanx style="verb">item_extension_data</spanx>.
+        </t>
+        <t>
+          <spanx style="verb">item_extensions</spanx> is a vector of 0 or more item extensions. This vector MUST NOT include more than one extension with the same <spanx style="verb">item_extension_type</spanx>. The extensions in the vector MUST be ordered by the value of the <spanx style="verb">item_extension_type</spanx> field, smallest value first.
+        </t>
+      </section>
+      <section title="Merkle Tree Leaves" anchor="tree_leaves">
+        <figure>
+          <preamble>
+            The leaves of a log's Merkle Tree correspond to the log's entries (see <xref target="log_entries"/>). Each leaf is the <xref target="mht">leaf hash</xref> of a <spanx style="verb">TransItem</spanx> structure of type <spanx style="verb">x509_entry</spanx> or <spanx style="verb">precert_entry</spanx>, which in this version (v2) encapsulates a <spanx style="verb">TimestampedCertificateEntryDataV2</spanx> structure. Note that leaf hashes are calculated as HASH(0x00 || TransItem), where the hashing algorithm is specified in the log's metadata.
+          </preamble>
+          <artwork>
     opaque TBSCertificate&lt;1..2^24-1&gt;;
 
     struct {
+        uint64 timestamp;
         opaque issuer_key_hash[HASH_SIZE];
         TBSCertificate tbs_certificate;
-    } CertInfo;
-
+        SctExtension sct_extensions&lt;0..2^16-1&gt;;
+    } TimestampedCertificateEntryDataV2;</artwork>
```

Comment:
Between ";" and "</artwork"?
The document consistently does _not_ include a newline there, so that "xml2rfc --html" doesn't add an unnecessary blank line at the bottom of each artwork.